docs/essentials/numericsincore.md

```diff
@@ -1 +1,58 @@
+# Numerics in .NET Core
 
+.NET Core supports the standard numeric integral and floating-point primitives, as well as [System.Numerics.BigInteger](http://dotnet.github.io/api/System.Numerics.BigInteger.html), an integral type with no theoretical upper or lower bound, [System.Numerics.Complex](http://dotnet.github.io/api/System.Numerics.Complex.html), a type that represents complex numbers, and a set of Single Instruction Multiple Data (SIMD)-enabled vector types in the [System.Numerics](http://dotnet.github.io/api/System.Numerics.html) namespace. 
```

Comment:
Would it make sense to link SIMD to [its Wikipedia page](https://en.wikipedia.org/wiki/SIMD)?